libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/offroute/OffRouteDetector.java

```diff
@@ -209,7 +209,7 @@ private static boolean movingAwayFromManeuver(RouteProgress routeProgress,
                                                 List<Point> stepPoints,
                                                 Point currentPoint) {
 
-    if (routeProgress.currentLegProgress().upComingStep() == null || stepPoints.isEmpty()) {
+    if (routeProgress.currentLegProgress().upComingStep() == null || stepPoints.size() < 2) {
```

Comment:
Same comments as above in `NavigationMapRoute` regarding this `if` check

Reply:
Sorry I am not 100% sure what to do here :). Should I extract the 2 into a variable or convert the conditions into variable?

The line length looks alright, so I am not sure, if this is about shortening the line.

Reply:
@boldtrn yep, exactly what you did with https://github.com/mapbox/mapbox-navigation-android/pull/1108/files#diff-cce5da198b8d3c30274778f11300d70bR448

Also replacing the `2` with `TWO_POINTS` as well.

Reply:
Thanks for the clarification, I changed the code accordingly.